pxtlib/docsrender.ts

```diff
@@ -434,6 +438,12 @@ namespace pxt.docs {
                 if (m) return `<li class="${m[1] == ' ' ? 'unchecked' : 'checked'}">` + text.slice(m[0].length) + '</li>\n'
                 return '<li>' + text + '</li>\n';
             }
+            renderer.link = function (href: string, title: string, text: string) {
+                const relative = href.indexOf('/') == 0;
+                const target = !relative ? '_blank' : '';
+                if (relative && d.versionPath) href = `/${d.versionPath}${href}`;
+                return `<a href="${href}" ${title ? `aria-label="${title}"` : ''} ${target ? `target="${target}"` : ''}>${text}</a>`;
```

Comment:
are these strings escaped already?

Reply:
This comment looks good: https://github.com/markedjs/marked/issues/655#issuecomment-383226346

Reply:
Good find. I'll use that comment instead and patch in my fix.